web/src/app/schedules/ScheduleCalendar.js

```diff
@@ -241,27 +238,27 @@ export default class ScheduleCalendar extends React.PureComponent {
     )
   }
 
-  getCalEvents = (shifts, _fixedShifts) => {
-    const fixedSchedules = _fixedShifts.map((sched) => ({
+  getCalEvents = (shifts, _tempScheds) => {
+    const TempSchedules = _tempScheds.map((sched) => ({
```

Comment:
nit:
```suggestion
    const tempSchedules = _tempScheds.map((sched) => ({
```
and references